core/app/models/spree/payment_method.rb

```diff
@@ -7,7 +7,7 @@ class PaymentMethod < Spree::Base
 
     scope :active,                 -> { where(active: true) }
     scope :available,              -> { active.where(display_on: [:front_end, :back_end, :both]) }
-    scope :available_on_front_end, -> { active.where(display_on: [:front_end, :both]) }
+    scope :available_on_front_end, -> { active.where(display_on: [:front_end, :both]).order(position: :asc) }
```

Comment:
yes i agree

to order the available scope makes more sense though.


On 11.08.2016 19:56, Vishal Zambre wrote:
> >      scope :available,              -> { active.where(display_on: [:front_end, :back_end, :both]) }